client/task-list/tasks/Marketing/Plugin.scss

```diff
@@ -0,0 +1,42 @@
+.woocommerce-plugin-list__plugin {
+	display: flex;
+	padding: $gap-large;
+	border-top: 1px solid #e5e5e5;
```

Comment:
Should we be using a color variable, maybe `$table-border` (which is `#e2e4e7) or `$gray-200` (which is `#e0e0e0`)?

/cc @elizaan36

Reply:
Yep I believe that would be `$gray-200`

Reply:
Switched this to `$gray-200` in https://github.com/woocommerce/woocommerce-admin/pull/7419.